ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml

```diff
@@ -2,6 +2,13 @@ ref:
   as: openshift-e2e-test
   from: tests
   commands: openshift-e2e-test-commands.sh
+  env:
+  - name: TEST_COMMAND
```

Comment:
This unfortunately doesn’t solve the common “run a type of suite like upgrade and then run the normal suite”.

I would recommend not abstracting the test command into multiple steps, just accept a test command like we do in CI chunks today.  I don’t think this is going to address our needs well.

Reply:
No worries, we can  still iterate ;)